src/homeWork13/SortAlgorithms.java

```diff
@@ -0,0 +1,30 @@
+package homeWork13;
+
+public class SortAlgorithms {
+
+    public static int[] insertionSort(int[] array) {
+        for (int i = 1; i < array.length; i++) {
+            int j = i;
```

Comment:
why `j = i`? It should point to the previous element - `j = i - 1`, as it doesn't make sense to compare the same elements